### ISSUE TITLE
[Feature Request] Option to hide the Steam Big Picture mode button

### ISSUE THREAD

**Issue by Telescuffle** [purogamer/Fluent-for-Steam#33]
I personally feel the Big Picture mode button is too prominent within the UI - even though I would assume it is rarely purposely used by end-users. Additionally, the iconography using in the Fluent skin doesn't really reflect the function of the button. 

My suggestion is to add an option to hide this button or move it to somewhere less prominent in the UI.

Apart from that, love the skin :)

**Comment by purogamer** [comment]
![image](https://user-images.githubusercontent.com/87723899/163734172-1182f267-bd5f-40d7-bbb3-6f4518e44b6d.png)
new 

future new location for the big picture, in a much more natural place what do you think?

**Comment by purogamer** [comment]
![image](https://user-images.githubusercontent.com/87723899/163734199-e5e647c9-fba2-4093-bff7-20dbfa633dca.png)
preview of new overlay


![image](https://user-images.githubusercontent.com/87723899/163734213-7f17e946-cafd-479f-a406-499c6e78639e.png)
the old overlay will still exist in parallel but will have a revision in some margins. here the sample

**Comment by Telescuffle** [comment]
Yup, I think that looks good and the other UI changes also sound good. 

Look forward to getting it in testing :)

On an unrelated note, what are you using to give Windows a MacOS vibe? E.g. The Dock and the bar at the top

**Comment by purogamer** [comment]
> Yup, I think that looks good and the other UI changes also sound good. 
> 
> Look forward to getting it in testing :)
> 
> On an unrelated note, what are you using to give Windows a MacOS vibe? E.g. The Dock and the bar at the top

thanks for the contribution. 

On the other hand, I am using Windows 11 and an application called mydockfinder, I discovered it recently and I thought it was great because it uses Windows libraries. That results in it having a lot of compatibility with windows. Besides that it is stable and has many options, if you want you can buy it on steam.

**Comment by Telescuffle** [comment]
Awesome! Will have to check it out on one of my devices! :D
